docs/core-concepts/bot-sequences/actions.mdx

```diff
@@ -1,9 +1,624 @@
+
+**Example: Typing "Hello"**
+
+```json
+{
+    "name": "Type 'Hello'",
+    "description": "Types 'Hello' using the keyboard",
+    "endCriteria": [...],
+    "botAction": {
+        "type": "InputPlayback",
+        "data": {
+            "startTime":0.0,
+            "inputData": {
+                "keyboard": [
+
+                    {"startTime":0.0,"binding":"<Keyboard>/leftShift"},
+                    {"startTime":0.0,"endTime":0.0,"binding":"<Keyboard>/h"},
+                    {"endTime":0.0,"binding":"<Keyboard>/leftShift"},
+
+                    {"startTime":0.0,"endTime":0.0,"binding":"<Keyboard>/e"},
+                    {"startTime":0.0,"endTime":0.0,"binding":"<Keyboard>/l"},
+                    {"startTime":0.0,"endTime":0.0,"binding":"<Keyboard>/l"},
+                    {"startTime":0.0,"endTime":0.0,"binding":"<Keyboard>/o"}
+
+                ],
+                "mouse":[]
+            }
+        }
+    }
+}
+```
+
+**Example: Moving and clicking the mouse**
+
+```json
+{
+    "name": "Move and click the mouse",
+    "description": "Moves the mouse to a position and clicks it",
+    "endCriteria": [...],
+    "botAction": {
+        "type": "InputPlayback",
+        "data": {
+            "startTime": 0.0,
+            "inputData": {
+                "keyboard": [],
+                "mouse": [
+                    {"startTime":0.0,"screenSize":{"x":1920,"y":1080},"position":{"x":942,"y":500},"worldPosition":null,"leftButton":false,"middleButton":false,"rightButton":false,"forwardButton":false,"backButton":false,"scroll":{"x":0,"y":0},"clickedObjectNormalizedPaths":["UI Canvas/Start Button"]},
+                    {"startTime":0.1,"screenSize":{"x":1920,"y":1080},"position":{"x":942,"y":500},"worldPosition":null,"leftButton":true,"middleButton":false,"rightButton":false,"forwardButton":false,"backButton":false,"scroll":{"x":0,"y":0},"clickedObjectNormalizedPaths":["UI Canvas/Start Button"]}
+                ]
+            }
+        }
+    }
+}
+```
+
+### Behaviour
+
+The `Behaviour` action type runs a MonoBehaviour. This can be useful for running agent code that relies on
+particular data and information from the code of your game, such as tapping into an existing navigation system.
+
+**Data Format**
+
+`type` is set to `Behaviour`.
+
+```json
+// The `botAction.data` format
+{
+    "behaviourFullName": "...", // The full class path of the MonoBehaviour to run
+    "maxRuntimeSeconds": ... // The maximum amount of time to run the behaviour for in seconds before it is stopped
+}
+```
+
+The Behaviour itself needs to be defined in the codebase of your game where it can be found during runtime. It should be
+a class that inherits from `MonoBehaviour`. In order to terminate the behaviour, you can use `Destroy(this)` to destroy the
+behaviour instance.
+
+**Example: Running a pathfinding behaviour for at max 30 seconds**
+
```

Comment:
We need to add a blurb about IRGBotSegmentActionBehaviour to make Behaviours pausible.


We also need to add a short blurb explaining that the Update loop of your behaviour will run as normal until you destroy it or the timer limit is reached at which point the SDK will destroy it